scripts/health_check.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+from enum import Enum
+
+import requests
+
+
+ENV_ENUM = Enum(
+    'ENV',
+    [
+        ('dev', 'https://addons-dev.allizom.org'),
+        ('stage', 'https://addons.allizom.org'),
+        ('prod', 'https://addons.mozilla.org'),
+        # TODO: maybe we could use the local environmnet here
+        ('test', ''),
```

Comment:
I would remove all the special `test` bits here and below and just use:
```suggestion
        # For local environments hit the nginx container as set in docker-compose.yml
        ('local', 'http://nginx'),
```